scripts/west_commands/runners/nrfutil.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''Runner for flashing with nrfutil.'''
+
+import json
+import os
+import sys
+import subprocess
+import tempfile
+
+from runners.nrf_common import NrfBinaryRunner
+
+ErrNotAvailableBecauseProtection = 24
+ErrVerify = 25
```

Comment:
shouldn't these be imported?

Reply:
Where from? The only source for those numbers is `nrfutil device list-error-codes --json-pretty`

Reply:
Ah sorry, yes, you are right. Will fix.